### PR TITLE
bnd cli improvements

### PR DESCRIPTION
**Pull request by stbischof**
- use getSourcePath (not getAllsourcepath) when compiling
- use include/exclude globs for parallel build.
- test parallel build
- removed single project itest in par (@pkriens )
- test transitive dependencys
- remove full option in buildoptions
- forker print infos to out not to error stream
- formatter

### CODE REVIEW

File: biz.aQute.bnd/test/aQute/bnd/main/TestBndMain.java [hunk 1]
```diff
@@ -218,7 +259,7 @@ public void testBuild() throws Exception {
 
 		capturedStdIO.getSystemErrContent()
 			.contains(
-			"000: p: The JAR is empty: The instructions for the JAR named p did not cause any content to be included, this is likely wrong");
+			EMPTY_JAR_MSG);
```
Comment (pkriens) [inline biz.aQute.bnd/test/aQute/bnd/main/TestBndMain.java]: in general we can do `assertTrue(processor.check("The JAR is empty:"));` The given strings in the parameters are filtered from the errors and warnings. If anything remains or a string is not found false is returned.

Comment (bjhargrave) [reply biz.aQute.bnd/test/aQute/bnd/main/TestBndMain.java]: We also need to get rid of test which assert that out/err is empty. When run on Java 9+, we get can get warnings from the JVM about illegal access issue which have nothing to do with the test but the test fails anyway.

**Comment by stbischof** [comment]
updated!
- fix windows linebreak issues
- can ignore warnings in System.err
- tests expected errors
